src/api/todos.ts

```diff
@@ -0,0 +1,19 @@
+import { NewTodo } from '../types/NewTodo';
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const postTodo = (data: NewTodo) => {
+  return client.post<Todo>('/todos', data);
+};
+
+export const deleteTodo = (id: number) => {
+  return client.delete(`/todos/${id}`);
+};
+
+export const updateTodo = (id: number, data: object) => {
```

Comment:
Instead of using 'object' as a type, you should use existing todo interface with Partial